p2p/message: Changed evidence message to contain evidence, not a list… (#394)
tendermint/spec: spec/p2p/messages/evidence.md

## Message Types

### Evidence

> Verified evidence that has already been propagated throughout the network. This evidence can appear within an EvidenceList within the [block](../../core/data_structures.md#block) as well.

Reviewer: ```suggestion
Verified evidence that has already been propagated throughout the network. This evidence will appear within the EvidenceList struct of a [block](../../core/data_structures.md#block) as well.
```
nit: to use more assertive language
